### DISCUSSION TITLE
Add new compartments

### DISCUSSION THREAD

**Discussion by ggshalala** [Open-Systems-Pharmacology/Forum#1586] [Questions & Problems]
Hello, everybody!
I have a question for you. I have added three compartmens S1, S2 and S3 to the kidney in the MoBi spatial structure and set relevant parameters, but the simulation results show that the drug simulation curve in the compartments S1, S2 and S3 is 0. I would like to know why this is and who can help me?
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/141638628/fdc14358-8a16-4d68-bcfa-16b5b6fc185f)

**Comment by Christoph27** [comment]
There are several potential reasons, hard to say without further details of what you have done. 
Possible reasons are e.g.: 
- Missing neighborhoods between your new compartments and the previously present one.
- The passive transport process which is supposed to be responsible for getting drug  into the new compartments to not recognize the new compartments as target.

For the concepts you could look at: https://docs.open-systems-pharmacology.org/working-with-mobi/mobi-documentation/building-block-concepts
